comp/trace/config/testdata/full.yaml

```diff
@@ -91,3 +91,4 @@ apm_config:
       luhn: true
     cache:
       enabled: true
+      max_size: 5555555
```

Comment:
why this value is not equal to `5000000` ?

Reply:
Looks env var is prioritized. I added a test case that expects `o.Cache.MaxSize` equals to `5555555`.